src/components/Editor/CallSite.css

```diff
@@ -4,19 +4,20 @@
 
 .call-site {
   background: #f0f9ff;
+  border-bottom: 2px solid #aed3ef;
   position: relative;
 }
 
 .call-site::before {
-  content: "";
-  position: absolute;
+  content: ">>";
   width: 100%;
   height: calc(100% - 2px);
   border-bottom: 2px solid #aed3ef;
 }
 
 .call-site-bp {
   position: relative;
+  border-bottom: 2px solid red;
```

Comment:
I'm guessing this is for debugger?

Reply:
Not completely sure I understand the question, but the red border is the current way of indicating that a breakpoint is selected. I added it here so that both the icon and text continue to be underlined.

Reply:
Ahh, I see.  We probably want to find a different color, but that's a simple change so don't hang up on it.